corgi/collectors/errata_tool.py

```diff
@@ -225,3 +233,30 @@ def get_builds_for_errata(self, errata_id: int) -> list[int]:
             for component in variant:
                 build_ids.update(component.keys())
         return list(build_ids)
+
+    def _parse_module(
+        self,
+        module_name: str,
+        module_data: dict[str, dict[str, list[str]]],
+        brew: Brew,
+        variant_to_component_map: typing.DefaultDict[str, list],
+    ):
```

Comment:
If this is returning None, let's specify that:
```suggestion
    ) -> None:
```

Reply:
updated